lldb/include/lldb/API/SBStructuredData.h

```diff
@@ -111,6 +111,7 @@ class SBStructuredData {
 protected:
   friend class SBAttachInfo;
   friend class SBLaunchInfo;
+  friend class SBPlatform;
   friend class SBDebugger;
   friend class SBTarget;
   friend class SBProcess;
```

Comment:
We keep friending new classes in all of the `lldb::SB` header files. I wonder if we should create a single class inside of lldb that has all of the abilities to access the private methods of any `lldb::SB` classes and then have a bunch of static calls on this one class that everyone internally can use?

Reply:
That would be nice, but I think we should do this in a different patch. I'll keep in mind though 🙂